src/Symfony/Component/Form/Extension/Core/Type/DateType.php

```diff
@@ -320,6 +321,7 @@ public function configureOptions(OptionsResolver $resolver): void
         $resolver->setAllowedTypes('months', 'array');
         $resolver->setAllowedTypes('days', 'array');
         $resolver->setAllowedTypes('input_format', 'string');
+        $resolver->setAllowedTypes('calendar', ['null', \IntlCalendar::class]);
```

Comment:
Could you also add some brief info about this new option? `$resolver->setInfo('calendar', '...')`